### PR TITLE
Allow Amorphous characters to pass through barred window frames

### PR DESCRIPTION
**Pull request by NoddingCrow**
#### Summary

SUMMARY: Balance "Amorphous trait characters can pass through barred window frames"

#### Purpose of change

The Amorphous trait's unique interaction with metal bars terrain (t_bars) is currently extremely niche and almost never used outside of police buildings. With this change, an unencumbered Amorphous character can perform the same ability on an _empty_ windowframe with metal bars (t_window_bars), giving them a little more use from their ability and improving consistency of behaviour with visually identical tiles.

#### Describe the solution

Apply the examine-action 'bars' to t_window_bars, t_metal_grate_window and t_metal_grate_window_noglass.

#### Describe alternatives you've considered

Also applying this to other permeable objects e.g. chain link fences, though they already have the ability to be vaulted over so I consider them less important.

#### Testing

Characters with and without Aqueous/Amorphous spawned and used to examine t_bars, t_window_bars and the derivatives (alarm, curtain, etc).
Only the empty bars - no glass - support the slime movement as desired. No phasing through glass! Options successfully do not present to characters without the trait. Dialogue box correctly pulls the name of the tile.

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
There are several types of windows with metal grating (t_metal_grate_window_noglass) which could also benefit from this.

File: data/json/furniture_and_terrain/terrain-windows.json [hunk 1]
```diff
@@ -557,6 +557,7 @@
     "name": "window frame with metal bars",
     "looks_like": "t_bars",
     "description": "A giant sheet of glass inserted into a window with thick security grilles, making it impossible to crawl through.  Typically installed for high-value stores, or at least stores in bad neighborhoods.",
+    "examine_action":"bars",
```
Comment (BrettDong) [inline data/json/furniture_and_terrain/terrain-windows.json:560]:
```suggestion
    "examine_action": "bars",
```

**Review by BrettDong — CHANGES_REQUESTED** [review]
Have to follow the JSON style convention in this repository to pass CI checks. You can run `make style-json` to format JSON files before making a commit.

**Comment by NoddingCrow** [comment]
Okiedoke! 
@Night-Pryanik - I agree, and have added the action to the grated window and grated window noglass too (since they're functionally identical and neither have glass in them. Weird.) 
The ones further down the tree, the heavily grated windows with curtains on, I'm gonna make an executive decision and say they're too densely packed to slime through. Gotta give 'em some disadvantage, much as I love the idea.

**Review by BrettDong — APPROVED** [review]

**Pull request merged**